master/internal/api_workspace.go

```diff
@@ -501,6 +513,33 @@ func (a *apiServer) PostWorkspace(
 	}, nil
 }
 
+func (a *apiServer) DeleteWorkspaceNamespaceBindings(ctx context.Context,
+	req *apiv1.DeleteWorkspaceNamespaceBindingsRequest,
+) (*apiv1.DeleteWorkspaceNamespaceBindingsResponse, error) {
+	currUser, _, err := grpcutil.GetUser(ctx)
+	if err != nil {
+		return nil, err
+	}
+	clusterNames := []string{}
+	for _, c := range req.ClusterNames {
+		newClusterName, err := a.validateRequestClusterName(c)
+		if err != nil {
+			return nil, err
+		}
+		if c != newClusterName {
+			clusterNames = append(clusterNames, newClusterName)
+		} else {
+			clusterNames = append(clusterNames, c)
+		}
```

Comment:
nit: Can we instead do 
```suggestion
		c = newClusterName
		clusterNames = append(clusterNames, c)
```

Reply:
done